### PR TITLE
Added shell sort in c++

### PR DESCRIPTION
**Pull request by tahmid02016**

### CODE REVIEW

**Review by ankitapuri — CHANGES_REQUESTED** [review]
explain your code with comments

**Comment by tahmid02016** [comment]
> explain your code with comments

Thank you for your valuable suggestion. I have added some comments describing 
the code. Please check now.

File: Shell_Sort/Shell_sort.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+			// j is the correct position for tmp
+			// so assign tmp to arr[j]
+			arr[j] = tmp;
+		}
+	}
+}
+
+// driver code
+int main () {
+	int arr [7] = {3, 2, 7, 5, 11, 9, 13}; // array of size 7
```
Comment (ankitapuri) [inline Shell_Sort/Shell_sort.cpp]: Kindly take input from user please

**Review by ankitapuri — CHANGES_REQUESTED** [review]

File: Shell_Sort/Shell_sort.cpp (hunk 1 above)
Comment (tahmid02016) [reply Shell_Sort/Shell_sort.cpp]: Done!

**Review by ankitapuri — APPROVED** [review]

**Pull request merged**